docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,111 @@
+    - **Peer TLS certificates**
+    - **Peer organization admin certificates (organization MSP)**
+    - **Peer enrollment certificates (peer local MSP)**  
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, must users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **Peer TLS certificate and private key**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `peer.tls.rootcert.file` parameter in `core.yaml`.
+        - Copy the TLS CA root certificate `ca-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca-cert.pem`. The path and name of the certificate corresponds to `peer.tls.clientRootCAs.files` parameter in `core.yaml`. Only required if mutual authentication (`clientAuthRequired`) is set to true.
+        - After you have generated the peer TLS certificate, rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
+        - Copy the peer TLS certificate and private key to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls`. The path and name of these certificates correspond to the `peer.tls.cert.file` and `peer.tls.key.file` parameters in the `core.yaml`.
+      - **Peer organization admin MSP**:
+        - After you have registered the peer organization admin user and enrolled the identity, copy the generated MSP folder to `organizations/peerOrganizations/org1.example.com/msp`. This is the peer organization MSP definition and is used by every peer node in the organization.
```

Comment:
organizations/peerOrganizations/org1.example.com/msp is for the channel org level msp.
What is being configured here is the peer's local MSP - organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp.
Same with the next bullet... the config.yaml that specifies the admin node ou goes in the peer's local msp directory. 
For this reason, I would make the admin config as the last bullet. That is, you need to define the peer identity (enrollment cert) in the local msp as per the next bullet, and then verify the admin config for that identity.

Reply:
Hopefully the edits I made address this as well.